cmd/deploy.go

```diff
@@ -529,6 +530,17 @@ func validateBuildType(buildType string) error {
 	return nil
 }
 
+// ErrInvalidBuilder indicates that the passed builder was invalid.
+type ErrInvalidBuilder error
+
+// ValidateBuilder validatest that the input Build type is valid for deploy command
+func validateBuilder(buildType string) error {
+	if errs := fn.ValidateBuilder(buildType, false); len(errs) > 0 {
```

Comment:
If the signature changes for the function in function_builder.go, then this helper function doesn't seem to be necessary is it?